src/component/category-form/category-form.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import autoBind from '../../utils/index';
+
+// this is in the UI STATE!!!!
+const defaultState = {
+  title:'',
+};
+
+class CategoryForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = this.props.category || defaultState;
+    autoBind.call(this, CategoryForm);
+  }
+  // ----------------MEMBER FUNCTIONS------------------
+  handleChange(event) {
+    const { value } = event.target;
+    this.setState({ title: value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.props.onComplete(this.state);
+    this.setState(defaultState);
+  }
+
+  // --------------------LIFECYCLE HOOKS-------------------
+
+  static getDerivedStateFromProps(nextProps) {
```

Comment:
remove this function, it makes your `this.state` alway display your default state